perftestnotebook/analyzer.py

```diff
@@ -41,8 +41,23 @@ def get_header(self):
             template_header_content = f.read()
             return template_header_content
 
-    def get_notebook_section(self, func):
+    def get_notebook_section(self, function_name,function_parameter=None):
+        notebook_section = ""
         template_function_folder_path = "testing/resources/notebook-sections/"
-        template_function_file_path = os.path.join(template_function_folder_path, func)
+        template_function_file_path = os.path.join(template_function_folder_path, function_name)
         with open(template_function_file_path, "r") as f:
-            return f.read()
+            line = f.readline()
+            while line:
+                if function_name in line and function_parameter is not None:
+                    parameter_string = ""
+                    for (x,y) in list(function_parameter.items()):
```

Comment:
x,y isn't very informative. You should use `var_name` and `var_value` or something like that.